lib/p2p/Peer.ts

```diff
@@ -82,6 +88,10 @@ class Peer extends EventEmitter {
   private readonly responseMap: Map<string, PendingResponseEntry> = new Map();
   private connectTime!: number;
   private connectionRetriesRevoked = false;
+  /** The version of xud this peer is using. */
+  private _version?: string;
+  /** The node pub key of this peer. */
+  private _nodePubKey?: string;
```

Comment:
Why do these new variables have an underscore as prefix?

Reply:
I made getter functions for these two variables and needed to use different naming for the variables themselves, this was the cleanest way I could think to do it. It was done as part of #1055. I can change it in a separate PR if you have a better approach in mind.